src/assets/scripts/client/airport/AirportWindModel.js

```diff
@@ -0,0 +1,95 @@
+import _floor from 'lodash/floor';
+import { calculateNormalDistributedNumber } from '../math/core';
+
+export default class AirportWindModel {
+    /**
+     * Constructor. Sets up the class with
+     * invalid properties.
+     * 
+     * @constructor
+     */
+    constructor(data) {
+        this.angle = -9999;
+        this.speed = -1;
+
+        return this._init(data);
+    }
+
+    /**
+     * Method to initialize default wind values.
+     * 
+     * @method _init
+     * @private
+     */
+    _init(data) {
+        this.speed = data.speed;
+        this.angle = data.angle;
+
+        const nextWind = {
+            speed: this.speed,
+            angle: this.angle
+        };
+
+        return this._calculateNextWind(nextWind);
```

Comment:
the `nextWind` arg here is not needed.  you could simply call the method.  This would overwrite any default values, though.  Is that intended?

Reply:
So, what I'm trying to get my head around is airport-specific wind defaults: so, just pass the data to `_init`, and overwrite the defaults. If it isn't passed a `speed` or `angle` KV-pair, it should just leave it as default.

Reply:
ahh, ok.

First question to ask there is wind required to be defined in the airport file?  You'd have to check the docs for that answer.  If it is, then there may not be a need to call `_calculateNextWind()` here.  you can simply set the data and be done.  You may need to translate the data, though.  hmm 🤔